### PR TITLE
[Merged by Bors] - Kiln

### PR DESCRIPTION
**Pull request by paulhauner**
## Issue Addressed

Adds the [Kiln](https://github.com/eth-clients/merge-testnets/tree/main/kiln) configs, so we can use `--network kiln`. 

## Additional Notes

- Also includes the fix from #3066.

### CODE REVIEW

File: common/eth2_network_config/built_in_network_configs/kiln/boot_enr.yaml [hunk 1]
```diff
@@ -0,0 +1 @@
+- enr:-Iq4QMCTfIMXnow27baRUb35Q8iiFHSIDBJh6hQM5Axohhf4b6Kr_cOCu0htQ5WvVqKvFgY28893DHAg8gnBAXsAVqmGAX53x8JggmlkgnY0gmlwhLKAlv6Jc2VjcDI1NmsxoQK6S-Cii_KmfFdUJL2TANL3ksaKUnNXvTCv1tLwXs0QgIN1ZHCCIyk
```
Comment (michaelsproul) [inline common/eth2_network_config/built_in_network_configs/kiln/boot_enr.yaml:1]: Looks like there are two more bootnodes in the README: https://github.com/eth-clients/merge-testnets/tree/main/kiln#kiln-testnet

Comment (paulhauner) [reply common/eth2_network_config/built_in_network_configs/kiln/boot_enr.yaml:1]: Nice spotting, added in c45e6291a.

**Review by michaelsproul — APPROVED** [review]
LGTM.

Spun up a node and it found 26 peers, even with the single bootnode.

**Comment by paulhauner** [comment]
Thank you!

bors r+

**Pull request closed without merging**